code/src/cljs/sixsq/nuvla/ui/intercom/views.cljs

```diff
@@ -0,0 +1,21 @@
+(ns sixsq.nuvla.ui.intercom.views
+  (:require
+    [re-frame.core :refer [dispatch subscribe]]
+    [sixsq.nuvla.ui.utils.intercom :as intercom]
+    [sixsq.nuvla.ui.intercom.subs :as subs]
+    [sixsq.nuvla.ui.session.subs :as session-subs]
+    [sixsq.nuvla.ui.main.subs :as main-subs]
+    [taoensso.timbre :as log]))
+
+(defn widget
+  []
+  (fn []
+    (let [_      (subscribe [::main-subs/nav-path])
+          app-id (subscribe [::subs/app-id])
+          email  (subscribe [::session-subs/user])
+          events (subscribe [::subs/events])]
+      @_
```

Comment:
What is this symbol? Typo?

Reply:
To force the execution of the function, AFAIK we have to deref the values, otherwise re-frame is smart enough no to execute it.